rabbitmq/config.go

```diff
@@ -0,0 +1,51 @@
+package rabbitmq
+
+import "fmt"
+
+type RabbitMqConf struct {
+	Username string
+	Password string
+	Host     string
+	Port     int
+	VHost    string `json:",optional"`
+}
+
+type RabbitMqListenerConf struct {
+	RabbitMqConf
+	ListenerQueues []ConsumerConf
+}
+
+type ConsumerConf struct {
+	Name      string
+	AutoAck   bool `json:",default=true"`
+	Exclusive bool `json:",default=false"`
+	NoLocal   bool `json:",default=false"` // Set to true, which means that messages sent by producers in the same connection cannot be delivered to consumers in this connection
+	NoWait    bool `json:",default=false"` // Whether to block processing
+}
+
+type RabbitMqSenderConf struct {
+	RabbitMqConf
+	ContentType string // MIME content type
```

Comment:
Can we put a default value here?

Reply:
> Can we put a default value here?

You can set the default value `text/plain`。Other optional values, such as `application/json`, `application/xml` and other MIME types

Reply:
I mean:

ContentType string `json:",default=text/plain"`

Reply:
> I mean:
> 
> ContentType string `json:",default=text/plain"`

ok